substrate/frame/nomination-pools/src/adapter.rs

```diff
@@ -262,7 +264,8 @@ impl<T: Config, Staking: StakingInterface<Balance = BalanceOf<T>, AccountId = T:
 		pool_account: Pool<Self::AccountId>,
 		_: Member<Self::AccountId>,
 	) -> BalanceOf<T> {
-		T::Currency::balance(&pool_account.0).saturating_sub(Self::active_stake(pool_account))
+		// free/liquid balance of the pool account.
```

Comment:
`balance()` contains the frozen part which means can be not liquid

Reply:
Technically this works because pool will never have frozen funds (its a pool generated pot account) but is confusing. I will fix it.